pkg/providers/aws/provider_postgres.go

```diff
@@ -806,11 +808,11 @@ func (p *PostgresProvider) exposePostgresMetrics(ctx context.Context, cr *v1alph
 	resources.SetMetricCurrentTime(resources.DefaultPostgresInfoMetricName, infoLabels)
 
 	// set available metric
-	if instance == nil || *instance.DBInstanceStatus != "available" {
+	if len(string(cr.Status.Phase)) == 0 || cr.Status.Phase != croType.PhaseComplete {
```

Comment:
Since this is an `available` metric I think we need to keep the check on line `809`. 
We should be checking the instance for this metric, and not the phase of the cr, so I don't think this should have to change

Reply:
`available` was implemented but never used in alerts see https://issues.redhat.com/browse/INTLY-6521?focusedCommentId=14095362&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14095362, as this jira was to check for the phase and not the aws Status I decided I would reuse this function.

Reply:
the `available` metric is used and alerted on https://github.com/integr8ly/integreatly-operator/blob/master/pkg/resources/metrics.go#L26
so is the `connectivity` metric https://github.com/integr8ly/integreatly-operator/blob/master/pkg/resources/metrics.go#L55
I meant the `info` metric was never used in alerts https://github.com/integr8ly/cloud-resource-operator/blob/master/pkg/providers/aws/provider_postgres.go#L806 

I think for postgres and redis providers all we need to do is update the status to have both `awsStatus` and `CR phase Status` in the info labels https://github.com/integr8ly/cloud-resource-operator/blob/master/pkg/providers/aws/provider_postgres.go#L764

Reply:
Thanks @ciaranRoche see what I was missing. moving phase status to its own function